tools/export/__init__.py

```diff
@@ -335,6 +336,9 @@ def export_project(src_paths, export_path, target, ide, libraries_paths=None,
         for _, res in resource_dict.iteritems():
             resources.add(res)
 
+    psa_files_dir = process_manifest_files(resources.psa_manifests, export_path)
```

Comment:
Does this need some detection logic too?

Reply:
please take a look at `tools/toolchains/__init__.py` in the modified files.
We have are utilizing existing scaning for finding our manifests and adding them as `resources.psa_manifests`

Reply:
Correct, you added the detection logic to [`build_project`](https://github.com/ARMmbed/mbed-os/pull/5828/files#diff-8ae3dde46b55f3a922724cd3585922b2R532). As export does not run `bulid_project` it will not have the same check run. As this is the same logic, it should be guarded by the same check.

Reply:
done